### PR TITLE
Adding Details on Testing responsibilities + Test plan template

### PR DESCRIPTION
**Pull request by asos-arieldan**
## General information

- Jira ticket link:

- Changelog:
  - After discussing with @asos-benschofield and @asos-stuartcrocker we identified the need for clearer boundries around testing.
  - The addition of a test plan to contributing PRs is a mechanised way to raise awareness around testing activities and responsibilities
  - Added some details on the way testing should be approached for contributing teams

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -78,12 +89,23 @@ Please make sure you fill the PR template correctly, stating the Jira ticket, ch
 
 At this point you will need at least one approval from someone in your team before the QA process can start.
 
-After the first PR approval, a QA can begin the testing process. Following the agreed Test Plan your team's QA will test your feature. We can lend a test environment if required.
+After the first PR approval, a QA can begin the testing process.
+
+### Testing Activities
+
+Your team will be responsible of the vast majority of testing activities taking place for this increment. We will help you understand our test environments and lend them to you for your testing efforts as well as assist you in case you need help with our pipeline management.
```
Comment (asos-stuartcrocker) [inline CONTRIBUTING.md]: 'responsible for the vast majority...'

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -78,12 +89,23 @@ Please make sure you fill the PR template correctly, stating the Jira ticket, ch
...
+
+The test plan should that gets attached to the PR serve as a guidance on what types of testing needed and it is also a registry of the activities that were covered during this stage.
+
+Ask a QA engineer on our team to fill in knowledge gaps or to pair if needed, and make sure to run our team's automated test suit along with your manual efforts before signing off the PR.
+ 
+Increments to the automated test suite should be part of your team's PR.
 
 When your feature has been fully tested, you now need a second approval from our team to make sure everything still makes sense.
```
Comment (asos-stuartcrocker) [inline CONTRIBUTING.md]: I don't know what 'fully tested' means :)

Maybe - 'When you believe that you've completed all of the testing activities necessary...'

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -78,12 +89,23 @@ Please make sure you fill the PR template correctly, stating the Jira ticket, ch
...
 At this point you will need at least one approval from someone in your team before the QA process can start.
 
-After the first PR approval, a QA can begin the testing process. Following the agreed Test Plan your team's QA will test your feature. We can lend a test environment if required.
+After the first PR approval, a QA can begin the testing process.
+
+### Testing Activities
+
+Your team will be responsible of the vast majority of testing activities taking place for this increment. We will help you understand our test environments and lend them to you for your testing efforts as well as assist you in case you need help with our pipeline management.
+
+The test plan should that gets attached to the PR serve as a guidance on what types of testing needed and it is also a registry of the activities that were covered during this stage.
```
Comment (asos-stuartcrocker) [inline CONTRIBUTING.md]: 'plan that gets attached to the PR should serve as a guide to the types of testing required' 

btw the test plan was not meant to be a registry of the detailed activities completed, especially where exploratory testing sessions have taken place, it's more a 'testing contract' saying that one side will commit to this as a minimum bar. This is why I've started introducing the session reports to allow for a more detailed appraisal of the testing activities completed.

Comment (asos-arieldan) [reply CONTRIBUTING.md]: Fair point

File: CONTRIBUTING.md (hunk 1 above)
Comment (asos-arieldan) [reply CONTRIBUTING.md]: Prepositions are a nightmare for me! :) fixing

File: CONTRIBUTING.md (hunk 2 above)
Comment (asos-arieldan) [reply CONTRIBUTING.md]: 👍
